components/resources/library/build.gradle.kts

```diff
@@ -122,12 +132,21 @@ kotlin {
             dependsOn(skikoTest)
             dependsOn(blockingTest)
         }
-        val jsMain by getting {
+        val webMain by creating {
             dependsOn(skikoMain)
         }
+        val jsMain by getting {
+            dependsOn(webMain)
+        }
+        val wasmJsMain by getting {
```

Comment:
We need to keep a mirroring of main and test source sets to avoid hidden problems.